src/main/java/frc/robot/subsystems/Reaper.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.Solenoid;
+
+public class Reaper extends SubsystemBase {
+
+    // ESC declarations
+    private final VictorSP reaperMotor;
+
+    // Piston declarations
+    private final Solenoid portDeployPiston;
+    private final Solenoid starboardDeployPiston;
+
+    // class initializer
+    public Reaper(VictorSP intakeMotor, int portDeployPistonPort, int starboardDeployPistonPort) {
+        this.reaperMotor = intakeMotor;
+        portDeployPiston = new Solenoid(portDeployPistonPort);
+        starboardDeployPiston = new Solenoid(starboardDeployPistonPort);
+    }
+
+    // Setters
+
+    /**
+     * 
+     * @param power positave intakes ball? negative outakes the balls? duhhhh
+     */
+    public void reaperMotorControl(double power) {
```

Comment:
I don't love the "motor control" name. I prefer verbs for functions. This could be `runReaper`